rules/windows/builtin/win_mal_creddumper.yml

```diff
@@ -1,35 +1,60 @@
 ---
 action: global
-title: Malicious Service Install
-description: This method detects well-known keywords of malicious services in the Windows System Eventlog 
-author: Florian Roth
+title: Credential dumping tools service execution
+description: Detects well-known credential dumping tools execution via service execution events
+author: Florian Roth, Teymur Kheirkhabarov, Daniil Yugoslavskiy, oscd.community
+date: 2017/05/15
+modified: 2019/11/01
+references:
+    - https://www.slideshare.net/heirhabarov/hunting-for-credentials-dumping-in-windows-environment
 tags:
     - attack.credential_access
+    - attack.execution
     - attack.t1003
+    - attack.t1035
     - attack.s0005
+detection:
+    selection_1:
+        - ServiceName|contains:
+            - 'fgexec'
+            - 'wceservice'
+            - 'wce service'
+            - 'pwdump'
+            - 'gsecdump'
+            - 'cachedump'
+            - 'mimikatz'
+            - 'mimidrv'
+        - ImagePath|contains:
+            - 'fgexec'
+            - 'dumpsvc'
+            - 'cachedump'
+            - 'mimidrv'
+            - 'gsecdump'
+            - 'servpw'
+            - 'pwdump'
+        - ImagePath|re: '((\\\\.*\\.*|.*\\)([{]?[0-9A-Fa-f]{8}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{12}[}])?\.(exe|scr|cpl|bat|js|cmd|vbs).*)'
+    condition: selection and selection_1
+falsepositives:
+    - Legitimate Administrator using credential dumping tool for password recovery
+level: high
+---
 logsource:
     product: windows
     service: system
 detection:
-    selection1:
-        EventID: 
-          - 7045
-    keywords:
-        Message:
-          - '*WCE SERVICE*'
-          - '*WCESERVICE*'
-          - '*DumpSvc*'
-    quarkspwdump:
```

Comment:
"quarkspwdump" section deleted to make this rule detect only service executions. new rule to replace this section created — [win_quarkspwdump_clearing_hive_access_history.yml](https://github.com/Neo23x0/sigma/blob/1f7b3bc9a2d4c1f743da51f4283547d741b3c0e7/rules/windows/builtin/win_quarkspwdump_clearing_hive_access_history.yml):


```
title: QuarksPwDump clearing access history
status: experimental
description: Detects QuarksPwDump clearing access history in hive
author: Florian Roth
date: 2017/05/15
tags:
  - attack.credential_access
  - attack.t1003
level: critical
logsource:
    product: windows
    service: system
detection:
    selection:
        EventID: 16
        HiveName: '*\AppData\Local\Temp\SAM*.dmp'
    condition: selection
falsepositives:
    - Unknown
```

which basically the same with what was inside original rule.